pilot/pkg/xds/bootstrapds.go

```diff
@@ -0,0 +1,61 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package xds
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+
+	bootstrapv3 "github.com/envoyproxy/go-control-plane/envoy/config/bootstrap/v3"
+	discovery "github.com/envoyproxy/go-control-plane/envoy/service/discovery/v3"
+	"github.com/golang/protobuf/jsonpb"
+
+	"istio.io/istio/pilot/pkg/model"
+	"istio.io/istio/pilot/pkg/networking/util"
+	"istio.io/istio/pkg/bootstrap"
+)
+
+// Bootstrap generator produces an Envoy bootstrap from node descriptors.
+type BootstrapGenerator struct {
+	Server *DiscoveryServer
+}
+
+var _ model.XdsResourceGenerator = &BootstrapGenerator{}
```

Comment:
can we add to https://github.com/istio/istio/blob/932c3454878f09670f34063c59a7461bebe8954c/pilot/pkg/xds/v3/model.go#L46 to make the logs less noisy?

Reply:
Done.